testing/esp/esdlcmd/baselines/wstest-cpp-installdir/WsTestServiceBase.cpp

```diff
@@ -0,0 +1,527 @@
+
+#include "WsTestServiceBase.hpp"
+#include "jliball.hpp"
+#include "jlog.hpp"
+#include "jptree.hpp"
+
+extern "C" int onWsTestAllVersionReport(const char* CtxStr, const char* ReqStr, StringBuffer& RespStr)
+{
+    Owned<WsTestServiceBase> service = createWsTestServiceObj();
+    return service->AllVersionReport(CtxStr, ReqStr, RespStr);
```

Comment:
It seems odd to return the 64 bit address of an allocated buffer as a 32 bit integer.